docroot/themes/humsci/su_humsci_theme/su_humsci_theme.theme

```diff
@@ -113,6 +113,13 @@ function su_humsci_theme_preprocess_node(&$vars) {
   }
 }
 
+/**
+ * Implements hook_preprocess_paragraph().
+ */
+function su_humsci_theme_preprocess_paragraph(&$vars) {
+  $vars['#attached']['library'][] = 'su_humsci_theme/' . $vars['paragraph']->bundle();
```

Comment:
Perhaps you should validate that the library exists. What happens when the next developer that is unaware of this code creates a type and not a library for it?

Reply:
nothing happens. there's internal processes somewhere that will only add the library if it exists. It doesn't even throw anything in the dblogs